classes/03class/exercises/c03-docker01/drii-cavalcanti/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.7
+RUN apk add terraform
```

Comment:
The solution is functional, Adriana. That's good, but your image has more than 80MB, so something else is in the final image.

#### Can you make the final docker image size be less than 80MB?

Terraform itself is around 50MB, Alpine is around 5MB.

**Tip**: Terraform comes in a zipped file, that just needs to be unzipped and moved to a folder that is in the PATH environment variable. This should result in less than 60MB.

You can find terraform here: https://www.terraform.io/downloads.html

Reply:
am I supposed to use the wget command?

Reply:
Not necessarily.

One option is go with a multistage build:
https://docs.docker.com/develop/develop-images/multistage-build/

Or you can use wget to download the zip file and then remove all non necessary files from the build(like wget, zip tool and the terraform.zip file)...
Hit me up if you need further explanation.

Reply:
also, make sure you have the right version of terraform...:)